python/syntax.lark

```diff
@@ -69,7 +69,7 @@ hex_pattern: /[0-9a-fA-F]+/
 escape_code: /[nrtv0fb]/
 tupledef: "(" types ("," types)+ (",")? ")"
 recorddef: "{" (record_entry_lit (";" | "\n"))* record_entry_lit (";" | "\n")? "}"
-spread: ".." NAME
+spread: ".." expression
```

Comment:
Interesting! So `[..[1] + [2]]` is valid

Nothing wrong with that though; JS also allows that:

```js
[...5645645664n + '']
```